ui/raidboss/data/02-arr/raid/t5.js

```diff
@@ -62,42 +47,42 @@
     },
     {
       id: 'T5 Fireball',
-      regex: / 15:\y{ObjectId}:Twintania:5AC:Fireball:\y{ObjectId}:(\y{Name}):/,
-      regexDe: / 15:\y{ObjectId}:Twintania:5AC:Feuerball:\y{ObjectId}:(\y{Name}):/,
-      regexFr: / 15:\y{ObjectId}:Gémellia:5AC:Boule de feu:\y{ObjectId}:(\y{Name}):/,
-      regexJa: / 15:\y{ObjectId}:ツインタニア:5AC:ファイアボール:\y{ObjectId}:(\y{Name}):/,
+      regex: Regexes.ability({ source: 'Twintania', id: '5AC' }),
+      regexDe: Regexes.ability({ source: 'Twintania', id: '5AC' }),
+      regexFr: Regexes.ability({ source: 'Gémellia', id: '5AC' }),
+      regexJa: Regexes.ability({ source: 'ツインタニア', id: '5AC' }),
       alertText: function(data, matches) {
-        if (data.me == matches[1]) {
+        if (data.me == matches.target) {
           return {
             en: 'Fireball on YOU',
           };
         }
       },
       infoText: function(data, matches) {
-        if (data.me != matches[1]) {
+        if (data.me != matches.target) {
           return {
-            en: 'Fireball on ' + data.ShortName(matches[1]),
+            en: 'Fireball on ' + data.ShortName(matches.target),
```

Comment:
I do really wonder if we need a "thing on X" style response, as this is a common pattern.  However, I'm not sure how to handle the translations on that.  This is just a side thought, and not something that needs to get done in this pull request.

Reply:
It might make things easier for translation? Since everything follows a similar pattern, `on X` for English, `sur X`, `auf X`, `→ X`, etc.
When in doubt, you could use the full name of the ability for initial translation passes.

Reply:
Yeah, we could just use `matches.ability`, pretty easily.